aw_transform/merge_events_by_keys.py

```diff
@@ -13,22 +13,14 @@ def merge_events_by_keys(events, keys) -> List[Event]:
         return events
     merged_events = {}  # type: Dict[str, Event]
     for event in events:
-        summed_key = ""
-        for key in keys:
-            if key in event.data:
-                summed_key = summed_key + "." + event["data"][key]
+        summed_key = "/".join(event['data'][key] for key in keys)
         if summed_key not in merged_events:
             merged_events[summed_key] = Event(
                 timestamp=event.timestamp,
                 duration=event.duration,
-                data={}
+                data=event.data
```

Comment:
Not sure about the implications of this, but should work fine?

Reply:
This just create odd and unneeded data IMO.
What value does it have to know the data of the first event in a series of merged events? Just makes things confusing and increases the size of the data.

Reply:
I remember the reason for adding it now. I wanted to merge events by URL and still have a title in the merged event, I could add the title to the keys, but that would create way more events if one URL can have many different titles (which is generally uninteresting, as it's usually just prepending a `(1) Facebook` or similar notification indicator).

Maybe add a parameter `keep_data` to enable the behavior?

Reply:
And that's IMO a reason why we should avoid using titles for grouping as they are unreliable. One time when you do the transform you might get the title `Johan wrote you a message - Messenger` and another time just `Messenger` on the same exact data just because the order gets messed up.

If you actually care about the title you should IMO either:
- group by title as well
- do something similar to our previous more advanced "tree-like" version of chunking where you could see the chunked events other values chunked as well (but we removed that before because it generated a lot of data and was overly complex, so probably not a good idea)
- Add a optional argument which can enable giving a list of non-merged keys and for example set the title key in data to a list of all titles in the merged events (I think this might be the most clean solution). This is a nice example of where optional arguments fits in better than a new function as the base data is still the same, it's just an option to extend the empty data field.

Reply:
You have a point but those options are all significantly less performant than my suggested solution/workaround. 

I'm assuming that `merge_events_by_keys` plays a somewhat significant role in the performance of many queries so I'd rather not do a more expensive operation than necessary.

Reply:
I just don't feel comfortable maintaining a feature that is broken by design. For the current use-cases we have it doesn't impact performance as it's an option.